Dockerfile

```diff
@@ -52,6 +53,7 @@ RUN dpkg --add-architecture i386 \
         jq \
         python3-minimal \
         python3-pkg-resources \
+    && localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8 \
```

Comment:
What error are we talking about? For me locale-gen just produces:
```
Generating locales (this might take a while)...
  en_US.UTF-8... done
Generation complete.
```
and that's it?

Reply:
It was failing when I built the image locally.  I'll switch my local repo back to the `main` branch and build to locate the exact error message once I finish testing my latest changes.

Reply:
Ok let me know. Because all `locale-gen` is doing is calling `localedef` so we only need the one or the other. But as far as I can tell it's already good. There used to be a locale error in a very very old version, like three weeks back or so.

Reply:
Oh could it be that you executed commands before `locale-gen`? That's when you would see errors because the locale doesn't exist yet.

Reply:
Huh I can't seem to repro this anymore (was seeing it consistently earlier somehow??).  I'll remove this change for now and just keep an eye out to see if it happens again.